src/style.css

```diff
@@ -1 +1,136 @@
-/* empty source */
+html {
+  --main-text-colour: rgba(6, 11, 53, 1);
+  --second-text-colour: rgba(97, 96, 112, 1);
+  --button-colour: rgba(0, 172, 220, 1);
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--main-text-colour);
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  background: white;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  color: var(--main-text-colour);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.card__product-code {
+  color: var(--second-text-colour);
+  font-size: 10px;
+  line-height: 14px;
+  margin: 0 0 16px;
+}
+
+.card__review-row {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 0 24px;
+}
+
+.stars {
```

Comment:
Follow the checklist 
![image](https://github.com/mate-academy/layout_product-cards-en/assets/99403198/c259d36b-4c84-4995-91b1-73791b0d8ced)